packages/common/src/convert.ts

```diff
@@ -0,0 +1,307 @@
+import type { Multibase } from 'multiformats';
+
+import { base58btc } from 'multiformats/bases/base58';
+import { base64url } from 'multiformats/bases/base64';
```

Comment:
thoughts on using `paulmillr`'s [`scure-base`](https://github.com/paulmillr/scure-base) for base encoding? makes sense to use multiformats given that we've already pulled it in as a dependency. if we do decide to switch to `scure-base` we'll want to use `base64urlnopad` ([context](https://github.com/paulmillr/scure-base/issues/4))

Reply:
Great idea -- created an issue to track follow-up https://github.com/TBD54566975/web5-js/issues/153